openedx/core/djangoapps/certificates/api.py

```diff
@@ -19,6 +27,66 @@ def _enabled_and_instructor_paced(course):
     return False
 
 
+def certificates_viewable_for_course(course):
```

Comment:
This is replacing the code in may_certify(), right?

Reply:
Yup.